behave_test_automation/pah_behave/features/steps/logout_view.py

```diff
@@ -0,0 +1,32 @@
+from behave import when, step, then
+import features.steps.selectors as selector
+from features.steps.common_steps import short_wait
+
+
+@when('User clicks on hamburger menu')
+def click_hamburger_menu(context) -> None:
+    hamburger_menu = context.driver.find_element(*selector.hamburger_menu)
+    hamburger_menu.click()
+
+    short_wait(context.driver, selector.hamburger_menu_add_drive)
+    short_wait(context.driver, selector.hamburger_menu_drives)
+
+
+@step('User clicks logout button')
```

Comment:
I don't like "clicks" steps. Step should reflect business value and clicking buttons isn't meaningful.